modules/ROOT/pages/import-and-export-preferences-studio.adoc

```diff
@@ -0,0 +1,37 @@
+= Export and Import Studio Preferences
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can export your Anypoint Studio preferences and import them into a new version.
+
+Follow this process if you had to download a fresh copy of Studio and you want to recover your previous configurations.
+
+== Export Your Studio Configurations
+
+. In Studio, go to *Preferences*.
+. Ensure that you applied all the configurations that you want to export.
+. Select the *Export* icon image:export-icon.png[30,30]
++
+image::export-preferences.png[]
+. Configure the preferences that you want to export:
```

Comment:
Why "configure"? Aren't they already configured, and you just want to MOVE them?
Or maybe "Select only the configurations that you want to export"?